content/docs/2.0/scalers/metrics-api.md

```diff
@@ -0,0 +1,59 @@
++++
+title = "Metrics API"
+layout = "scaler"
+availability = "v2.0+"
+maintainer = "Community"
+description = "Scale applications based on a metric provided by an API"
+go_file = "metrics_api_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `metrics-api` trigger that scales based on a metric value provided by an API. 
+This scaler allows users to utilize any existing APIs as a metric provider.  
+
+**Parameter list:**
+- `url`: Full URL of the API operation to call to get the metric value (eg. `http://app:1317/api/v1/stats`).
+- `valueLocation`: [GJSON path notation](https://github.com/tidwall/gjson#path-syntax) to refer to the field in 
+    the payload containing the metric value
+- `targetValue`: Target value to scale on. When the metric provided by the API is equal or higher to this value, 
+    KEDA will start scaling out. When the metric is 0 or less, KEDA will scale down to 0.
+
+Here is an example of trigger configuration using metric-api scaler:
+
+```yaml
```

Comment:
Nit - This is typically listed before the parameters list; it's easier to resonate.